object-cache.php

```diff
@@ -457,6 +451,7 @@ function switch_to_blog( $blog_id ) {
 	function colorize_debug_line( $line ) {
 		$colors = array(
 			'get' => 'green',
+			'get_multi' => 'green',
```

Comment:
This change has since been merged via https://github.com/Automattic/wp-memcached/pull/53 instead.